apps/console/scripts/post-build.js

```diff
@@ -18,14 +18,21 @@
 
 const path = require("path");
 const fs = require("fs-extra");
+const DeploymentConfig = require("../src/public/deployment.config.json");
+
+let appFolder = DeploymentConfig.appBaseName;
+
+if (appFolder === "console") {
```

Comment:
Since we have the `PRE_AUTH_CHECK` in the environment, do we need to have this check?

Reply:
This was added since `scripts/post-build.js` has no access for the .env.local file to check the `PRE_AUTH_CHECK` variable.

Reply:
Yeah this worked. Fixed in the latest changes.